public/js/directives.js

```diff
@@ -71,9 +71,13 @@ angular.module('mean.directives', [])
       link: function(scope, elem, attr) {
         scope.showOptions = true;
 
-        if (scope.$$childHead.global.authenticated === true) {
-          scope.showOptions = false;
-        }
+        // if (scope.$$childHead.global.authenticated === true) {
+        //   scope.showOptions = false;
+        // }
```

Comment:
@codejockie please don't ask me this question. I've talked with you guys about this before